src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -427,6 +427,58 @@
             </div>
           </div>
 
+        <!-- Quotas -->
+        <div>
+          <legend i18n>Quotas</legend>
+
+          <!-- Max Bytes -->
+          <div class="form-group"
+               [ngClass]="{'has-error': form.showError('max_bytes', formDir)}">
+            <label class="control-label col-sm-3"
+                   for="max_bytes">
+              <ng-container i18n>Max bytes</ng-container>
+            </label>
+            <div class="col-sm-9">
+              <input class="form-control"
+                     id="max_bytes"
+                     name="max_bytes"
+                     type="text"
+                     formControlName="max_bytes"
+                     i18n-placeholder
+                     placeholder="e.g., 10GiB"
+                     defaultUnit="GiB"
+                     cdDimlessBinary>
+              <span class="help-block"
+                    i18n>Specify 0 to disable quota</span>
+              <span class="help-block"
+                    *ngIf="form.showError('max_bytes', formDir, 'required')"
+                    i18n>The value should be greater or equal to 0</span>
+            </div>
+          </div>
+
+          <!-- Max Objects -->
+          <div class="form-group"
+               [ngClass]="{'has-error': form.showError('max_objects', formDir)}">
+            <label class="control-label col-sm-3"
+                   for="max_objects">
+              <ng-container i18n>Max objects</ng-container>
+            </label>
+            <div class="col-sm-9">
+              <input class="form-control"
+                     id="max_objects"
+                     [min]="0"
+                     name="max_objects"
+                     type="number"
+                     formControlName="max_objects">
+              <span class="help-block"
+                    i18n>Specify 0 to disable quota</span>
```

Comment:
I wonder if we should allow the input to be empty.
Think we allow other places to leave it empty if we don't want to change its default value.

Reply:
Thanks, change behavior of these two new form control.

`quota_max_bytes` and `quota_max_objects` fields are
- not included: when fields are not dirty or user clear the fields
- included: when user enters valid values

Reply:
Since this is not related to validation I would remove this and add it beside the label, the same way that I used for the `max_bytes`.